cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/CxxCoverageSensor.java

```diff
@@ -282,13 +344,63 @@ private void checkLineCoverage(NewCoverage newCoverage, CoverageMeasure measure)
       }
     }
   }  
+
+  private static Set<File> getReportPaths(SensorContext context) {
+    Set<File> reportPaths = new HashSet<>();
+    Settings settings = context.settings();
+    FileSystem fs = context.fileSystem();
+
+    if (settings.hasKey(REPORT_PATHS_KEY)) {
+      for (String reportPath : settings.getStringArray(REPORT_PATHS_KEY)) {
+        File report = fs.resolvePath(reportPath);
+        if (!report.isFile()) {
+            LOG.info("Coverage report not found: '{}'", reportPath);
+        } else {
+          LOG.info("Add coverage report: '{}' '{}'", reportPath, report.getAbsolutePath());
+          reportPaths.add(report);
+        }
+      }
+    }
+
+    if (settings.hasKey(REPORT_PATH_KEY)) {
+      warnUsageOfDeprecatedProperty(settings, REPORT_PATH_KEY);
+      File report = fs.resolvePath(settings.getString(REPORT_PATH_KEY));
+      if (!report.isFile()) {
+        LOG.info("Coverage UT report not found: '{}'", settings.getString(REPORT_PATH_KEY));
+      } else {
+        reportPaths.add(report);
+      }
+    }
+
+    if (settings.hasKey(IT_REPORT_PATH_KEY)) {
+      warnUsageOfDeprecatedProperty(settings, IT_REPORT_PATH_KEY);
+      File report = fs.resolvePath(settings.getString(IT_REPORT_PATH_KEY));
+      if (!report.isFile()) {
+        LOG.info("Coverage IT report not found: '{}'", settings.getString(IT_REPORT_PATH_KEY));
+      } else {
+        reportPaths.add(report);
+      }
+    }
+
+    return reportPaths;
+  }
+
+  private static void warnUsageOfDeprecatedProperty(Settings settings, String reportPathProperty) {
+    if (!settings.hasKey(REPORT_PATHS_KEY)) {
+      LOG.warn("Property '{}' is deprecated. Please use '{}' instead.", reportPathProperty, REPORT_PATHS_KEY);
+    }
+  }
+  
   @Override
   protected String getSensorKey() {
     return KEY;
   }  
 
   @Override
   protected String reportPathKey() {
+    if (isSQ_6_2_or_newer) {
```

Comment:
Would change this for all versions to new naming. See no reason why we should be backward compatible as long there is a warning message.